core/tests/puppeteer-acceptance-tests/spec/blog-admin-tests.spec.js

```diff
@@ -0,0 +1,124 @@
+    blogAdmin = await userFactory.createNewBlogAdmin('blogAdm');
+    blogPostEditor = await userFactory.createNewBlogPostEditor(
+      'blogPostEditor');
+  }, DEFAULT_SPEC_TIMEOUT);
+
+  it('should assign roles to users and change tag properties',
+    async function() {
+      const guestUsr1 = await userFactory.createNewGuestUser(
+        'guestUsr1', 'guest_user1@example.com');
+      const guestUsr2 = await userFactory.createNewGuestUser(
+        'guestUsr2', 'guest_user2@example.com');
+
+      await superAdmin.expectUserNotToHaveRole('guestUsr1', ROLE_BLOG_ADMIN);
+      await blogAdmin.assignUserToRoleFromBlogAdminPage(
+        'guestUsr1', 'BLOG_ADMIN');
+      await superAdmin.expectUserToHaveRole('guestUsr1', ROLE_BLOG_ADMIN);
+
+      await superAdmin.expectUserNotToHaveRole(
+        'guestUsr2', ROLE_BLOG_POST_EDITOR);
+      await blogAdmin.assignUserToRoleFromBlogAdminPage(
+        'guestUsr2', 'BLOG_POST_EDITOR');
+      await superAdmin.expectUserToHaveRole('guestUsr2', ROLE_BLOG_POST_EDITOR);
+
+      await blogAdmin.removeBlogEditorRoleFromUsername('guestUsr2');
+      await superAdmin.expectUserNotToHaveRole(
+        'guestUsr2', ROLE_BLOG_POST_EDITOR);
+
+      await blogAdmin.expectTagToNotExistInBlogTags('Test_Tag');
+      await blogAdmin.addNewBlogTag('Test_Tag');
+      await blogAdmin.expectTagToExistInBlogTags('Test_Tag');
+
+      await blogAdmin.expectMaximumTagLimitNotToBe('5');
+      await blogAdmin.setMaximumTagLimitTo('5');
+      await blogAdmin.expectMaximumTagLimitToBe('5');
+      await guestUsr1.closeBrowser();
+      await guestUsr2.closeBrowser();
+    }, DEFAULT_SPEC_TIMEOUT);
+
+  it('should check blog dashboard accessible by blog admins',
+    async function() {
+      const guestUser = await userFactory.createNewGuestUser(
+        'guestUser', 'guest_user@example.com');
+
+      /** The blog-dashboard is not accessible to any guest user
+       *  as the user does not have the blog admin role. */
+      await guestUser.expectBlogDashboardAccessToBeUnauthorized();
+      await blogAdmin.expectBlogDashboardAccessToBeAuthorized();
+      await guestUser.closeBrowser();
+    }, DEFAULT_SPEC_TIMEOUT);
+
+  it('should create draft and delete draft blog post',
+    async function() {
+      await blogPostEditor.navigateToBlogDashboardPage();
+      await blogPostEditor.expectNumberOfBlogPostsToBe(0);
+      await blogPostEditor.createDraftBlogPostWithTitle('Test-Blog');
+
+      await blogPostEditor.expectNumberOfBlogPostsToBe(1);
+      await blogPostEditor.expectDraftBlogPostWithTitleToBePresent('Test-Blog');
+
+      await blogPostEditor.deleteDraftBlogPostWithTitle('Test-Blog');
+      await blogPostEditor.expectNumberOfBlogPostsToBe(0);
+    }, DEFAULT_SPEC_TIMEOUT);
+
+  it('should publish blog post and delete published blog post',
+    async function() {
+      pending('This test is giving errors. So excluding it right now.');
```

Comment:
What's going on here? This seems like a problem that needs to be looked into...

Reply:
This test is very much unpredictible and flaky. So it tends to fail most of the times. @Shivkant-Chauhan can address this in his PR #17354 after this gets merged.

Reply:
Still don't disable it, though. It should be run the standard way. I don't think that will block your PR since the CI action for the test isn't implemented yet.

Reply:
OK. Done